blog/2024-08-06-wasmcloud-1.1-brings-secrets-support-and-more-enhancements-for-enterprises/index.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: 'wasmCloud 1.1 brings secrets support and more enhancements for enterprises'
+image: './images/wasmcloud-1-1.png'
+authors: [brooksmtownsend]
+date: "2024-08-06"
+description: 'Learn about the new features in wasmCloud 1.1, including secrets support.'
+slug: wasmcloud-1.1-brings-secrets-support-and-more-enhancements-for-enterprises
+---
+
+![header](./images/wasmcloud-1-1.png)
+
+We're proud to announce the release of **wasmCloud 1.1**, including the feature most commonly requested by users: **secrets support**. In addition to secrets, the new release brings along a bevy of enhancements for enterprises, exciting new capabilities like Postgres, and first-class support for Go-based custom capability providers. Let's take a look at some of the highlights.
+
+{/* truncate */}
+
+## Secrets in wasmCloud
+
+Secrets are a must-have feature for enterprises, so it's no surprise that secret support has been our most-requested feature since the launch of 1.0. Now we're thrilled to provide full and flexible support for secrets with a secrets **backend system** that enables you to use the secret store of your choice while maintaining robust encryption across the lifecycle of a secret. 
```

Comment:
```suggestion
Secrets are a must-have feature for application platforms, so it's no surprise that secret support has been our most-requested feature since the launch of 1.0. Now we're thrilled to provide full and flexible support for secrets with a secrets **backend system** that enables you to use the secret store of your choice while maintaining robust encryption across the lifecycle of a secret. 
```